docs/_data/navigation.yml

```diff
@@ -1,109 +1,38 @@
-      - page: Configuration
-        url: /docs/get_started/configuration.html
-      - page: How to Deploy
-        url: /docs/get_started/how-to-deploy/overview.html
-        entries:
-        - page: Using Conjur
-          url: /docs/get_started/how-to-deploy/using-conjur.html
-        - page: Using Kubernetes Secrets
-          url: /docs/get_started/how-to-deploy/using-kubernetes-secrets.html
-  - title: Reference
-    url: /docs/reference.html
-    sections:
-      - page: Listeners
-        url: /docs/reference/listeners.html
-      - page: Handlers
-        url: /docs/reference/handlers/overview.html
-        entries:
-        - page: AWS - HTTPS
-          url: /docs/reference/handlers/http/aws.html
-        - page: Conjur - HTTP(S)
-          url: /docs/reference/handlers/http/conjur.html
-        - page: HTTP Basic Authentication
-          url: /docs/reference/handlers/http/basic.html
-        - page: MySQL
-          url: /docs/reference/handlers/mysql.html
-        - page: PostgreSQL
-          url: /docs/reference/handlers/postgres.html
-        - page: SSH Agent
-          url: /docs/reference/handlers/ssh_agent.html
-        - page: SSH
-          url: /docs/reference/handlers/ssh.html
-      - page: Credential Providers
-        url: /docs/reference/providers/overview.html
-        entries:
-        - page: CyberArk Conjur
-          url: /docs/reference/providers/conjur.html
-        - page: Environment
-          url: /docs/reference/providers/env.html
-        - page: File
-          url: /docs/reference/providers/file.html
-        - page: HashiCorp Vault
-          url: /docs/reference/providers/vault.html
-        - page: Keychain
-          url: /docs/reference/providers/keychain.html
-        - page: Literal
-          url: /docs/reference/providers/literal.html
-      - page: Configuration Managers
-        url: /docs/reference/config-managers/overview.html
-        entries:
-        - page: Configuration File
-          url: /docs/reference/config-managers/configfile.html
-        - page: Kubernetes CRD
-          url: /docs/reference/config-managers/k8s/crd.html
-      - page: Plugin Reference
-        url: /generated/pkg_secretless_plugin_v1.html
-      - page: Health Check
-        url: /docs/reference/health-check/overview.html
+footer-docs:
+  - title: Why Secretless?
+    url: https://docs.secretless.io/Latest/en/Content/Overview/WhySecretless.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Key Terms
+    url: https://docs.secretless.io/Latest/en/Content/Overview/key_terms.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Fundamentals
+    url: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
+    target: target="_blank" rel="noopener noreferrer"
+  - title: Plugin Reference
+    url: https://secretless.io/generated/pkg_secretless_plugin_v1.html
+    target: target="_blank" rel="noopener noreferrer"
+
+footer-res:
```

Comment:
in the interest of future clarity, it's a good idea to skip the abbreviation here and just call it `footer-resources`